src/components/experiment-tracking/run-metadata/run-metadata.js

```diff
@@ -9,10 +10,43 @@ import KebabIcon from '../../icons/kebab';
 import SelectedPin from '../../icons/selected-pin';
 import UnSelectedPin from '../../icons/un-selected-pin';
 import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { MetadataLoader } from './run-metadata-loader';
+import { experimentTrackingLazyLoadingColours } from '../../../config';
 
 import './run-metadata.css';
 import './animation.css';
 
+const Loader = ({ length, theme }) => {
+  const x = length > 1 ? 75 : 0;
+
+  return (
+    <tbody>
+      <tr>
+        <td>
+          <ContentLoader
+            viewBox="0 0 200 300"
+            width="200px"
+            height="100%"
+            backgroundColor={
+              theme === 'dark'
+                ? experimentTrackingLazyLoadingColours.backgroundDarkTheme
+                : experimentTrackingLazyLoadingColours.backgroundLightTheme
+            }
+            foregroundColor={
+              theme === 'dark'
+                ? experimentTrackingLazyLoadingColours.foregroundDarkTheme
+                : experimentTrackingLazyLoadingColours.foregroundLightTheme
+            }
+            speed={2}
+          >
+            <MetadataLoader x={x} />
+          </ContentLoader>
+        </td>
+      </tr>
+    </tbody>
+  );
+};
```

Comment:
Same comment here as my earlier one: I think it makes more sense to compose this in your loader file and then import and use it here.